src/assignment/modals/AddCollectionModal.tsx

```diff
@@ -325,31 +328,34 @@ const AddCollectionModal: FunctionComponent<AddCollectionModalProps> = ({
 					</ToolbarRight>
 				</Toolbar>
 
-				<Table
-					columns={tableColumns}
-					data={collections || undefined}
-					emptyStateMessage={
-						filterString
-							? t(
-									'assignment/modals/add-collection-modal___er-zijn-geen-collecties-die-voldoen-aan-de-zoekopdracht'
-							  )
-							: t(
-									'assignment/modals/add-collection-modal___er-zijn-nog-geen-collecties-aangemaakt'
-							  )
-					}
-					renderCell={(rowData: Avo.Collection.Collection, colKey: string) =>
-						renderCell(rowData, colKey as keyof Avo.Collection.Collection)
-					}
-					rowKey="id"
-					variant="styled"
-					onColumnClick={handleColumnClick as any}
-					sortColumn={sortColumn}
-					sortOrder={sortOrder}
-					useCards={isMobileWidth()}
-					showRadioButtons
-					selectedItemIds={selectedCollectionId ? [selectedCollectionId] : []}
-					onSelectionChanged={handleSelectedCollectionChanged}
-				/>
+				<ScrollBar>
+					<Table
+						columns={tableColumns}
+						data={collections || undefined}
+						emptyStateMessage={
+							filterString
+								? t(
+										'assignment/modals/add-collection-modal___er-zijn-geen-collecties-die-voldoen-aan-de-zoekopdracht'
+								  )
+								: t(
+										'assignment/modals/add-collection-modal___er-zijn-nog-geen-collecties-aangemaakt'
+								  )
+						}
+						renderCell={(rowData: Avo.Collection.Collection, colKey: string) =>
+							renderCell(rowData, colKey as keyof Avo.Collection.Collection)
+						}
+						rowKey="id"
+						variant="styled"
+						onColumnClick={handleColumnClick as any}
+						sortColumn={sortColumn}
+						sortOrder={sortOrder}
+						useCards={isMobileWidth()}
+						showRadioButtons
+						selectedItemIds={selectedCollectionId ? [selectedCollectionId] : []}
+						onSelectionChanged={handleSelectedCollectionChanged}
+					/>
+				</ScrollBar>
```

Comment:
Optional; use of this component can be avoided using a max-height on the modal-body and standard scrolling